manifests/config.pp

```diff
@@ -120,7 +120,7 @@
     $foreman_socket_override = template('foreman/foreman.socket-overrides.erb')
 
     if $foreman::ipa_authentication {
-      unless fact('foreman_ipa.default_server') and fact('foreman_ipa.default_realm') {
+      unless fact('foreman_ipa.default_realm') {
```

Comment:
given we don't need it anymore when we merge #967 -- what should we do here? ;-)

Reply:
Drop the whole `unless` block.